### PR TITLE
Add links to Gensim students ideas

### PR DESCRIPTION
**Pull request by rgaiacs**
@tmylk Could you provide the information request at the following comments? Thanks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,6 +70,19 @@ In alphabetic order.
     </td>
   </tr>
 
+  <tr>
+    <td>
+      <img width="300px" src=""/>
```
Comment (rgaiacs) [inline README.md]: Need logo of the project.

File: README.md [hunk 2]
```diff
@@ -70,6 +70,19 @@ In alphabetic order.
...
+    </td>
+    <td>
+       <h1>Gensim</h1>
+       <p>
+       </p>
```
Comment (rgaiacs) [inline README.md]: Need one line description.

File: organization/team.md [hunk 3]
```diff
@@ -32,6 +32,12 @@ organizing Google Summer of Code for NumFOCUS.
 | Irwin Zaid         | izaid           | izaid                     |
 | Mark Wiebe         | mwiebe          | mwiebe                    |
 
+#### Gensim
+
+| Name                 | Melange Profile | GitHub Profile            |
+|----------------------|-----------------|---------------------------|
+| Lev Konstantinovskiy |                 | tmylk                     |
```
Comment (rgaiacs) [inline organization/team.md]: Need the name of the others mentors.

File: README.md (hunk 1 above)
Comment (tmylk) [inline README.md]: http://radimrehurek.com/gensim/_static/images/logo-gensim.png

File: README.md (hunk 2 above)
Comment (tmylk) [inline README.md]: "Topic Modelling for Humans". Gensim is an open-source Python library for topic modelling, document indexing and similarity retrieval with large corpora. Target audience is the natural language processing (NLP) and information retrieval (IR) community. Memory-independent and distributed algorithms.

**Comment by rgaiacs** [comment]
@tmylk Thanks. Please let me know the email of the mentors so that I can invite them.

File: organization/team.md (hunk 3 above)
Comment (tmylk) [inline organization/team.md]: mentor Number Two:
Radim Rehurek
github: piskvorky

Comment (rgaiacs) [inline organization/team.md]: Done! Still need the email. Please send it to raniere@rgaiacs.com.

**Pull request merged**